README.md

```diff
@@ -71,17 +72,58 @@ make serve
+   - Subsocial - `src/chains/subsocial/config.ts`
+   - Soonsocial - `src/chains/soonsocial/config.ts`
 2. Necessary events/calls/storage calls can be configured for each chain in appropriate file in `/typegen` folder.
 3. Command `make typegen` uses custom [shel script](./scripts/typegen.sh) which generates types for each chain defined in `/typegen` folder.
-4. Chain sensitive logic is implemented in sub-folder with appropriate name in `/src/chains` folder. 
-It's required because each chain has it's own bunch of specs and as result it's own autogenerated types.
-Chain sensitive logic:
+4. Chain sensitive logic is implemented in sub-folder with appropriate name in `/src/chains` folder.
+   It's required because each chain has it's own bunch of specs and as result it's own autogenerated types.
+   Chain sensitive logic:
    - parsing events data
    - parsing calls data
    - making storage calls
 5. Indexer can be deployed to Aquarium to 2 different squids with 2 separate deployment manifests:
    - Subsocial - [./squid-subsocial.yaml](./squid-subsocial.yaml)
    - Soonsocial - [./squid-soonsocial.yaml](./squid-soonsocial.yaml)
-   
\ No newline at end of file
+
+## Search API
+
+Indexer GraphQL API has search query (`searchQuery`) with various parameters for filtering and full text search.
+
+### Search query arguments:
+
+- `indexes?: <string>` [ _default:_ `all` ] - _search within all or specific index (`all | spaces | posts`)_
+- `limit?: <number>` [ _default:_ `10` ] - _number of search hits per page._
+- `offset?: <number | null>` [ _default:_ `0` ] - _offset of search hits._
+- `q?: <string>` [ _default:_ `*` ] - _search query._
+- `spaceId?: <string>` - _filter search hits by provided `spaceId`._
+- `tags?: <string[]>` - _filter search hits by provided tags._
+
+All arguments listed above can be used together in any combination, except `spaceId + indexes:spaces`.
+
+### Search query results:
+
+- `err` - _search error if occurred. Can be `null` is not occurred._
+
+  - `reason: <string>` - _text message of occurred error._
+  - `status: <number>` - _error status code._
+
+- `hits` - _search results items list_.
+  - `_content` - _document source._
+    - `name: <string>` - _value of field `name` (actual only for Space entity)._
+    - `about: <string>` - _value of field `about` (actual only for Space entity)._
+    - `username: <string>` - _value of field `username` (actual only for Space entity)._
+    - `title: <string>` - _value of field `title` (actual only for Post entity)._
+    - `body: <string>` - _value of field `body` (actual only for Post entity)._
+    - `spaceId: <string>` - _value of field `spaceId` (actual only for Post entity)._
+    - `tags: <string[]>` - _list of tags._
+  - `_id: <string>` - _document ID (equal to on-chain entity ID)._
+  - `_index: <string>` - _index particular document is located in._
+  - `_score: <number>` - _search score of particular document._
```

Comment:
```suggestion
  - `_score: <number>` - _the search score of a particular document._
```
a particular document or this specific document?

Reply:
this specific document?